style.css

```diff
@@ -63,14 +61,22 @@ body {
     font-family: 'Roboto', sans-serif;
     font-size: 55px;
     flex-grow:1;
-    padding: 0px 0px 130px 150px;
+    padding-left: 150px;
 }
 
 .photo {
     flex-grow: 1; 
-    padding: 0px 0px 200px 0px; 
+    padding-bottom: 65px;
+    
 }
 .btn{
-    
-    border-radius: 50%;
-}
\ No newline at end of file
+    border-radius: 30px;
```

Comment:
radius is special, you can set percentage values in here 🙂

Reply:
ok :)